app/assets/stylesheets/application.scss

```diff
@@ -91,7 +91,7 @@ label {
 }
 
 .field_with_errors label, input:focus ~ label, input:valid ~ label {
-  top: -20px;
+  top: 0;
```

Comment:
Properties should be ordered color, font-size, top<br>Each selector in a comma sequence should be on its own single line